Develop

MaximMacarone opened the pull request:
INTEAM-5
INTEAM-8
INTEAM-9
INTEAM-14
INTEAM-16

MaximMacarone committed:
Impl notification system, Notification model, update .gitignore

MaximMacarone committed:
Move to xcodegen

MaximMacarone committed:
Update .gitignore

MaximMacarone committed:
Merge pull request #10 from iOS-vk-education/INTEAM-XCGEN

Move to xcodegen

MaximMacarone committed:
Merge pull request #9 from iOS-vk-education/INTEAM-9

Impl notification system, Notification model, update .gitignore

MaximMacarone committed:
Update README.md

MaximMacarone committed:
Update README.md

MaximMacarone committed:
info.plist change

MaximMacarone committed:
UserSheetView, project.yml fix

MaximMacarone committed:
Merge pull request #11 from iOS-vk-education/INTEAM-14

UserSheetView, project.yml fix

MaximMacarone committed:
Update README.md

MaximMacarone committed:
updated INotifier, notification example in AppointmentModel

MaximMacarone committed:
Merge pull request #12 from iOS-vk-education/INTEAM-16

updated INotifier, notification example in AppointmentModel

MaximMacarone committed:
Impl PetModel, Cat and Dog for example

MaximMacarone committed:
Merge pull request #13 from iOS-vk-education/INTEAM-5

Impl PetModel, Cat and Dog for example

MaximMacarone committed:
removed unneeded code

MaximMacarone committed:
Merge pull request #14 from iOS-vk-education/INTEAM-5

removed unneeded code

mleykhner committed:
Добавил протокол места и модель walkingAreaModel, сериализаторы к ней

mleykhner committed:
Merge remote-tracking branch 'origin/develop' into INTEAM-2
Догоняю develop, чтобы не было конфликтов)

mleykhner committed:
Я дурачок, не все файлы залил, исправляюсь

mleykhner committed:
Изменил конфигурацию xcodegen, включил в неё тесты

MaximMacarone committed:
aboba, changes to yml

Billynill committed:
In team 7.1 add UserModel (#16)

* first one

* Добавлена Модель пользователя

MaximMacarone committed:
Animation for settings view

mleykhner committed:
Сделал декодер чуть красивее

MaximMacarone committed:
Merge pull request #15 from iOS-vk-education/INTEAM-2

INTEAM-2: Добавил протокол места и модель walkingAreaModel, сериализаторы к ней

linlinich commented on README.md:
```diff
@@ -1,30 +1,32 @@
 ## 👨‍🏫 Менторы:
 
-* Ангеленина Решетникова - [linlinich](ub.com/linlinich)
+* Ангеленина Решетникова - [linlinich](https://github.com/linlinich)
```
Меня зовут Ангелина)

linlinich replied:
по-моему, так и осталось)

linlinich commented on lapki/Models/Notifications/NotificationCenter.swift:
```diff
@@ -0,0 +1,77 @@
+// This is the heart of notification system in this application.
+// Holds all incoming notifications and pushing them to
+// notification services that are user subscribed to.
+struct NotificationCenter {
```
Почему это структура? NotificationCenter же один на все приложение?

MaximMacarone replied:
сделано в класс

linlinich commented on lapki/Models/Pet/Pet.swift:
```diff
@@ -0,0 +1,73 @@
+enum CatBreed: String, CaseIterable {
+    case siamese
+    case persian
+    case bengal
```
Как эту историю расширять? А если у меня другая порода?

linlinich commented on lapki/Models/Pet/Pet.swift:
```diff
@@ -0,0 +1,73 @@
+}
+
+struct Dog: Pet {
+    typealias BreedType = DogBreed
```
зачем тут вообще замена? она не используется

MaximMacarone replied:
убрал

linlinich commented on lapki/Models/Pet/Pet.swift:
```diff
@@ -0,0 +1,73 @@
+}
+
+struct Cat: Pet {
+    typealias BreedType = CatBreed
```
тоже самое

MaximMacarone replied:
убрал

linlinich commented on lapki/Models/Pet/Pet.swift:
```diff
@@ -0,0 +1,73 @@
+    let id: String
+    var name: String
+    var age: Int
+    var breed: CatBreed
```
зачем разделять на котов и собак? а если у меня обезьяна? Тем более, поля одинаковые, кроме породы, но подход с отдельными породами мне не очень нравится

linlinich replied:
тут тоже не правили? Короче, не вижу смысла делать под котов и собак отдельные структуры. Для них никаких различий, кроме породы между собой нет. Для остальных экранов: записи, питомцы и тд нам не важно, какое у нас животное. предлоагаю эти структуры убрать, оставить только Pet, где будет enum с типом животного
предлагаю что-то типа:
```
enum BreedType: String, CaseIterable {
  case cat(CatBreed)
  case dog(DogBreed)
  case unknown //если у пользователя какое-то необычное животной
}
```

linlinich commented on lapki/Models/Pet/Pet.swift:
```diff
@@ -0,0 +1,73 @@
+            return Notification(title: "New notification from \(name)", body: "\(name) is hungry, feed them!", createdAt: Date.now)
+            case .wantsToWalk:
+            return Notification(title: "New notification from \(name)", body: "\(name) wants to walk, let's go!", createdAt: Date.now)
+        case .foodEnds:
```
поплыли отступы, некрасиво

linlinich commented on lapki/Models/Place/BusinessHours.swift:
```diff
@@ -0,0 +1,14 @@
+struct BusinessHours {
+    var openTime: Date?
+    var closeTime: Date?
+    var nonBusiness: Bool
```
нужно ли делать эти поля изменяемыми?

MaximMacarone committed:
Merge remote-tracking branch 'refs/remotes/origin/develop'

#Conflicts:
#	project.yml

linlinich commented on lapki/Models/Place/Photo.swift:
```diff
@@ -0,0 +1,16 @@
+typealias PhotoId = String
+
+struct PhotoItem: Codable {
+    let photo: PhotoId
```
ничего непонятно, но очень интересно. Не проще ли не делать отдельную сущность, и для фотоID просто хранить строку?
А так у тебя все равно итоговое значение - String, просто сначала в тайпэлиасе, а потом еще в структуре

linlinich replied:
let photo: String. не нужен typealias

linlinich commented on lapki/Models/Place/Place.swift:
```diff
@@ -0,0 +1,20 @@
+import CoreLocation
+
+protocol Place {
+    var id: Int { get set }
```
кажется, set на id не нужен.

linlinich commented on lapki/Models/Place/PlaceType.swift:
```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public enum PlaceType: String, Codable {
+    case clinic
```
такой же вопрос, как к породам. А если расширятся типы мест, станет больше значений?

linlinich commented on lapki/Models/Place/WalkingAreaModel.swift:
```diff
@@ -0,0 +1,28 @@
+struct WalkingAreaModel: Place, Codable {
+    
+    var isFavourite: Bool
+    
```
давай уберем энтеры между переменными, они тут ни к чему)

linlinich commented on lapki/Models/Place/WalkingAreaModel.swift:
```diff
@@ -0,0 +1,28 @@
+    
+    var isFavourite: Bool
+    
+    var coordinatesJson: String?
```
ты хочешь координаты хранить в json? Так не очень, я так понимаю, по этой структуре данных вы хотите отрисовывать экран. Нужно, чтобы при отрисовке экрана вы просто брали значение координат, а не дополнительно парсили еще

linlinich commented on lapki/Models/Place/WalkingAreaModel.swift:
```diff
@@ -0,0 +1,28 @@
+    
+    var coordinatesJson: String?
+    
+    var id: Int
```
id не должно меняться. Один раз в начале сгенерировалось и все

linlinich commented on lapki/Models/UserModel.swift:
```diff
@@ -0,0 +1,8 @@
+//  Created by 596 on 17.11.2024.
+//
+
+import Foundation
```
если в файле пока ничего нет, его и вливать не надо

linlinich commented on lapki/Models/Place/WeekBusinessHours.swift:
```diff
@@ -0,0 +1,10 @@
+
+import Foundation
+
+typealias WeekBusinessHours = [BusinessHours]
```
а зачем

linlinich commented on lapki/UserProfileViewController.swift:
```diff
@@ -0,0 +1,13 @@
+
+import Foundation
+struct User {
+    let id: UUID
```
там где-то видела еще один файл с User. надо убрать его тогда

MaximMacarone replied:
А точнее, было бы славно переместить содержимое этого файла в UserModel.swift, модели ViewController'ами не называют, а то запутаться можно

linlinich commented on lapki/Views/UserView/PetItem.swift:
```diff
@@ -0,0 +1,36 @@
+                .scaledToFit()
+                .frame(height: 40)
+            VStack(alignment: .leading) {
+                Text("Барсик")
```
если уже есть структура питомца, то нужно именно из нее прокидывать данные

linlinich commented on lapki/Views/UserView/UserSheet.swift:
```diff
@@ -0,0 +1,162 @@
+                            .frame(height: 32)
+                            .padding(.trailing, 8)
+                        VStack(alignment: .leading) {
+                            Text("Максим Лейхнер")
```
пока юзера сюда не прокидываем?

linlinich commented on lapki/Views/UserView/UserSheet.swift:
```diff
@@ -0,0 +1,162 @@
+                }
+                .padding(.horizontal, 24)
+                .shadow(color: .black.opacity(0.1), radius: 7)
+                //.frame(maxWidth: .infinity, maxHeight: 130)
```
давайте в мейн вливать без комментов

linlinich commented on lapki/Views/UserView/UserSheet.swift:
```diff
@@ -0,0 +1,162 @@
+                            .cornerRadius(24)
+                        }
+                        Button {
+                            
```
лишний энтер

linlinich commented on lapki/Views/UserView/UserViewController.swift:
```diff
@@ -0,0 +1,35 @@
+        
+        hostingController.didMove(toParent: self)
+    }
+    
+    
+
```
лишнее

linlinich commented on lapkiTests/lapkiTests.swift:
```diff
@@ -0,0 +1,40 @@
+            #expect(response.fencing == true)
+            #expect(response.lighting == false)
+            #expect(response.location.coordinates == [37.428639704, 55.747567311])
+            #expect(response.address == "Российская Федерация, город Москва, внутригородская территория муниципальный округ Кунцево, Рублёвское шоссе, дом 26, корпус 4")
```
лайк за тесты

mleykhner committed:
Добавлено переключение темы карты, отслеживание перемещения камеры

Для работы карты нужно создать файл lapki/Keys.plist. В него добавить API-ключ Яндекс Карт с тегом "YandexMapKey"

MaximMacarone committed:
UserView, UserProfileView, SettingsView, AppointmentsView, NotificationsView

MaximMacarone committed:
UserView, SettingsView, ProfileView, AppointmentsView, NotificationsView

MaximMacarone committed:
NotificationCenter struct -> class, services enum

MaximMacarone committed:
Merge pull request #19 from iOS-vk-education/INTEAM-9

NotificationCenter struct -> class, services enum

MaximMacarone committed:
Merge pull request #20 from iOS-vk-education/INTEAM-17

Inteam 17

MaximMacarone committed:
removed typealias

MaximMacarone committed:
Merge pull request #21 from iOS-vk-education/INTEAM-5

removed typealias

mleykhner committed:
Вернул навигацию, добавил делегат для менеджера карты

mleykhner committed:
Merge branch 'develop' into INTEAM-11

MaximMacarone committed:
changes to Notification system, working notification settings

mleykhner committed:
Добавлена карточка места, цветовые стили, подключена библиотека Glur

mleykhner committed:
Merge pull request #18 from iOS-vk-education/INTEAM-11

INTEAM-11

mleykhner committed:
Merge branch 'develop' into INTEAM-12

MaximMacarone committed:
Merge remote-tracking branch 'refs/remotes/origin/develop'

#Conflicts:
#	lapki/SceneDelegate.swift

MaximMacarone committed:
mapView+UserView

Billynill reviewed (approved):
'APPROVE' performed by Xcode.

Billynill reviewed (dismissed):
'APPROVE' performed by Xcode.

MaximMacarone committed:
bebra

mleykhner committed:
Merge remote-tracking branch 'origin/INTEAM-17' into INTEAM-12

savakotsur committed:
PlaceView + Place_MOCK + AppointmentModel fixes

savakotsur committed:
PlaceView + Place_MOCK + AppointmentModel fixes

mleykhner committed:
Изменил project.yml, добалил кложуры к карточке места

mleykhner committed:
Merge pull request #23 from iOS-vk-education/INTEAM-12

INTEAM-12

savakotsur committed:
Merge branch 'develop' into INTEAM-13

savakotsur committed:
Merge pull request #24 from iOS-vk-education/INTEAM-13

Inteam 13 + mock place + smth

savakotsur committed:
color fixes

savakotsur committed:
Added temporary view for PlaceView + some fixes

linlinich commented on lapki/Assets.xcassets/muhamed.imageset/IMG_1665.png:
какой мили

linlinich commented on lapki/Models/AppointmentModel.swift:
```diff
@@ -7,19 +7,29 @@
-    //var place: PlaceModel //Место (заготовка)
-    //var pet: PetModel //Питомец (заготовка)
+    var place: any Place //Место
+    var pet: any Pet //Питомец
```
короче мне не нравится, что к протоколу Pet нужно прописывать Any. Надо подумать, как этого избежать и поменять протокол

linlinich commented on lapki/Models/AppointmentModel.swift:
```diff
@@ -7,19 +7,29 @@
     let id: String = UUID().uuidString //Уникальный идентификатор
-    //var place: PlaceModel //Место (заготовка)
-    //var pet: PetModel //Питомец (заготовка)
+    var place: any Place //Место
```
тут any не нужен

linlinich commented on lapki/Models/Pet/PetManager.swift:
```diff
@@ -0,0 +1,18 @@
+    
+    private init() {}
+    
+    var MOCK_PETS: [any Pet] {
```
пока что у меня в голове не метчится архитектура прилы. Я так понимаю, это сделано для того, чтобы у нас была только одна сущность с питомцами, то есть минимизировать утечки и неточности при работе с данными. Из всех мест будем брать питомцев из этого класса? а зачем синглтон? в чем преимущество

linlinich commented on lapki/Utils/PawsColorSet.swift:
```diff
@@ -0,0 +1,49 @@
+
+extension Color {
+    enum Paws {
+        enum Background {
```
можете еще почитать про SwiftGen на будущее, генерит ресурсы автоматически (локализация, шрифты, цвета и тд)

linlinich commented on lapki/Views/AppointmentsView/AppointmentsView.swift:
```diff
@@ -0,0 +1,85 @@
+                VStack(spacing: 36) {
+                    VStack(spacing: 18) {
+                        ForEach(AppointmentManager.shared.MOCK_APPOINTMENTS) { appointment in
+                            AppointmentRowView(appointment: appointment)
```
по две для количества?

linlinich replied:
заведите в трекер задачу чтобы тут настроить корректое распределение, и чтобы на кнопку показать еще тоже норм данные показывались

linlinich commented on lapki/Views/AppointmentsView/AppointmentsView.swift:
```diff
@@ -0,0 +1,85 @@
+                        }
+                        .foregroundStyle(Color.Paws.Text.secondaryLabel)
+                        ForEach(AppointmentManager.shared.MOCK_APPOINTMENTS) { appointment in
+                            AppointmentRowView(appointment: appointment)
```
чисто вкидываю: было бы прикольно, если бы прошедшие записи бьли бы как-то заблюрены сереньким или что-то типа того, чтобы по ячейке было понятно, что запить прошла. Что-то такое имею в виду
![Снимок экрана 2024—12—11 в 21 30 04](https://github.com/user-attachments/assets/f8572b6a-8551-4713-9950-99251e9620b5)

linlinich commented on lapki/Views/AppointmentsView/Layouts/AppointmentRowView.swift:
```diff
@@ -0,0 +1,49 @@
+
+struct AppointmentRowView: View {
+    
+    var appointment: AppointmentModel
```
нам нужен var тут? Кажется, что мы ничего не меняем в модели

linlinich commented on lapki/Views/AppointmentsView/Layouts/AppointmentRowView.swift:
```diff
@@ -0,0 +1,49 @@
+            }
+            
+            VStack(alignment: .leading) {
+                Text("Красногвардейская ВЛ")
```
а зачем тут мок текст, если мы используем appointment? у которой есть название

linlinich commented on lapki/Views/AppointmentsView/Layouts/AppointmentRowView.swift:
```diff
@@ -0,0 +1,49 @@
+                    .scaledToFit()
+                    .frame(height: 40)
+            } else {
+                Image(systemName: "cat.circle")
```
дефолтную картинку хорошо, что задали, но не кошку бы, а какую-то для любого животного, например, лапку

linlinich commented on lapki/Views/NotificationsView/Layouts/NotificationItemView.swift:
```diff
@@ -0,0 +1,44 @@
+                .scaledToFit()
+                .frame(width: 40)
+            VStack(alignment: .leading, spacing: 3) {
+                Text("Корм почти закончился")
```
у notifiaction же есть поле title

linlinich commented on lapki/Views/NotificationsView/Layouts/NotificationItemView.swift:
```diff
@@ -0,0 +1,44 @@
+                    .font(.subheadline)
+                    .lineLimit(1)
+                    .foregroundStyle(Color.Paws.Text.label)
+                Text("Барсик")
```
это тоже должно лежать в модели и доставаться оттуда

linlinich commented on lapki/Views/NotificationsView/Layouts/NotificationItemView.swift:
```diff
@@ -0,0 +1,44 @@
+    
+    var body: some View {
+        HStack(alignment: .top, spacing: 6) {
+            Image(systemName: "cat.circle")
```
для картинки тоже нужна какая-то логика

linlinich commented on lapki/Views/NotificationsView/Layouts/NotificationItemView.swift:
```diff
@@ -0,0 +1,44 @@
+                    .foregroundStyle(Color.Paws.Text.secondaryLabel)
+            }
+            Spacer()
+            Text("13:27")
```
аналогично)

linlinich commented on lapki/Views/ParentView.swift:
```diff
@@ -0,0 +1,26 @@
+
+//#Preview {
+//    ParentView()
+//}
```
лучше тогда убрать вообще

linlinich commented on lapki/Views/PlaceCardView.swift:
```diff
@@ -0,0 +1,68 @@
+                                .font(.system(size: 20, weight: .medium, design: .rounded))
+                                .lineLimit(1)
+                                .foregroundStyle(Color.Paws.Constant.white)
+                            Text("Ветеринарное учреждение")
```
этот текст не должен быть замокан, замоканы должны быть данные в модельках, а во вью уже должны использоваться поля модели

linlinich replied:
я так поняла это пока сырой экран?

linlinich commented on lapki/Views/PlaceView/PlaceView.swift:
```diff
@@ -0,0 +1,161 @@
+                    .font(.system(size: 24, weight: .heavy, design: .rounded))
+                
+                HStack {
+                    Text("Ветеринарное учреждение") //place.type (нужно что-то сделать с тайпом для вывода имени тайпа)
```
задать для каждого значения enum значение на русском. Сейчас этот энам конформит протоколу string и в целом можно использовать place.type.rawValue
сейчас энам выглядит так:
```
public enum PlaceType: String, Codable {
    case clinic
    case walkingArea
    case vaccination
    case shelter
}
```
и его значения так и будут clinic и др.
нужно сделать что-то типа такого:
```
public enum PlaceType: String, Codable {
    case clinic = "Клиника"
    case walkingArea = "..."
    case vaccination = "..."
}
```
только вот у меня вопрос, откуда этот тип вообще берется? он парсится из json мест??

linlinich commented on lapki/Views/PlaceView/PlaceView.swift:
```diff
@@ -0,0 +1,161 @@
+                            Text("Телефон")
+                                .fontWeight(.medium)
+                                .padding(.bottom, 2)
+                            Text("+7 495 432 23 54") //не хватает номера телефона в модели
```
значит надо расширять модель, а здесь использовать только поля модели

savakotsur replied:
жду когда Макс поправит модель((

linlinich commented on lapki/Views/PlaceView/PlaceView.swift:
```diff
@@ -0,0 +1,161 @@
+                .padding(.bottom)
+                
+                // Description
+                Text("Структурное подразделение СББЖ ЮАО ГБУ «Мосветобъединение». Вакцинация, оформление ветеринарных сопроводительных документов, услуги по лечению животных. Работает центр лучевой диагностики и компьютерной томографии.") //не хватает дескрипшна в модели
```
расширяем значит

savakotsur replied:
Аналогично с телефоном

linlinich commented on lapki/Views/PlaceView/PlaceView.swift:
```diff
@@ -0,0 +1,161 @@
+                            .resizable()
+                            .padding(.vertical, 12)
+                            .scaledToFit()
+                        //                            .frame(width: 47, height: 47)
```
давайте лить без таких комментов, если это не комментарии, которые напоминания

savakotsur replied:
тут видимо просто забыл про это/не увидел

linlinich commented on lapki/Views/PlaceView/PlaceView.swift:
```diff
@@ -0,0 +1,161 @@
+            // Footer Buttons
+            HStack(spacing: 16) {
+                Button(action: {
+                    // Route action
```
вообще лучше писать такие комменты так:
// TODO: Route action
гораздо проще будет потом найти, что нужно доделать

linlinich commented on lapki/Views/SettingsView/SettingsView.swift:
```diff
@@ -0,0 +1,116 @@
+                    }
+                }
+                .padding(.horizontal, 16)
+#if DEBUG
```
красота

linlinich commented on lapki/Views/UserProfileView/Layouts/UserProfileHeaderView.swift:
```diff
@@ -0,0 +1,39 @@
+                .frame(width: 64, height: 64)
+                .clipShape(Circle())
+            VStack(alignment: .leading) {
+                Text("Константин Кулаков")
```
Где модель юзера потеряли? У нас все будут названы в честь Кости?)

MaximMacarone replied:
> Где модель юзера потеряли? У нас все будут названы в честь Кости?)

ДА

linlinich commented on lapki/Views/UserView/Layouts/PetItem.swift:
```diff
@@ -0,0 +1,45 @@
+                    }
+                    .shadow(color: Color.black.opacity(0.1), radius: 12)
+            } else {
+                Image(systemName: "cat.circle")
```
давай какую-то более универсальную картинку возьмем в качестве дефолтной

linlinich commented on lapki/Views/UserView/Layouts/PetItem.swift:
```diff
@@ -0,0 +1,45 @@
+                Text(pet.name)
+                    .font(.headline)
+                    .foregroundStyle(Color.Paws.Text.label)
+                Text("\(pet.age) года, \(pet.breed.rawValue)")
```
у меня чет эта строка выдает ошибку, с ней проект не компилится
из-за pet.breed.rawValue

linlinich commented on lapki/Views/UserView/Layouts/UserHeaderView.swift:
```diff
@@ -0,0 +1,64 @@
+                                .stroke(Color.Paws.Background.background, lineWidth: 3)
+                        }
+                        .shadow(color: Color.black.opacity(0.1), radius: 12)
+                        //.matchedGeometryEffect(id: "userIcon", in: userIconAnimation)
```
убираем

linlinich commented on lapki/Views/UserView/Layouts/UserHeaderView.swift:
```diff
@@ -0,0 +1,64 @@
+struct UserHeaderView: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var showUserProfileView: Bool
+    //var userIconAnimation: Namespace.ID
```
убираем

linlinich commented on lapki/Views/UserView/UserViewController.swift:
```diff
@@ -0,0 +1,35 @@
+//  Created by Maxim Makarenkov on 09.11.2024.
+//
+
+//import UIKit
+//import SwiftUI
+//
+//class UserViewController: UIViewController {
+//
+//    override func viewDidLoad() {
+//        super.viewDidLoad()
+//        
+//        let swiftUIView = ParentView()
+//        let hostingController = UIHostingController(rootView: swiftUIView)
+//        
+//        addChild(hostingController)
+//        hostingController.view.translatesAutoresizingMaskIntoConstraints = false
+//        view.addSubview(hostingController.view)
+//        
+//        NSLayoutConstraint.activate([
+//            hostingController.view.topAnchor.constraint(equalTo: view.topAnchor),
+//            hostingController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+//            hostingController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+//            hostingController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+//        ])
+//        
+//        hostingController.didMove(toParent: self)
+//    }
+//    
+//    
+//
```
такого точно не надо

linlinich commented on lapki/YandexMapManager.swift:
```diff
@@ -0,0 +1,114 @@
+                }
+            }
+        } else {
+            // TODO: Пока не знаю как сделать
```
какой молодец! лайк за туду

MaximMacarone replied:
Это намек на то, что надо доделать?

MaximMacarone committed:
Folder restructure, Pet struct change, deleted comments and unneded code

MaximMacarone committed:
еще че то

MaximMacarone committed:
.gitkeep added

MaximMacarone committed:
Мелкие исправления, code lint

MaximMacarone committed:
xcodegen update

MaximMacarone committed:
Working appointments list

MaximMacarone committed:
Merge pull request #27 from iOS-vk-education/INTEAM-22

INTEAM-22

MaximMacarone committed:
delete/update appointment

MaximMacarone committed:
Merge pull request #28 from iOS-vk-education/INTEAM-22

delete/update appointment

savakotsur committed:
some minor fixes with ui

MaximMacarone committed:
Add pet logic and views

MaximMacarone committed:
little change

MaximMacarone committed:
Merge branch 'develop' into INTEAM-24

MaximMacarone committed:
Merge pull request #29 from iOS-vk-education/INTEAM-24

INTEAM-24

mleykhner committed:
Добавил пины на карту, по нажатию на них открывается PlaceView

MaximMacarone committed:
Управление питомцами

MaximMacarone committed:
Merge branch 'develop' into INTEAM-29

MaximMacarone committed:
Merge pull request #30 from iOS-vk-education/INTEAM-29

Управление питомцами

MaximMacarone committed:
выезжающая штучка

MaximMacarone committed:
dog image

MaximMacarone committed:
Merge pull request #32 from iOS-vk-education/INTEAM-30

Inteam 30

MaximMacarone committed:
Merge pull request #31 from iOS-vk-education/map_pins

Добавил пины на карту, по нажатию на них открывается PlaceView

MaximMacarone committed:
pet creation fix

savakotsur committed:
microfixes

savakotsur committed:
microfixes

savakotsur committed:
added addingAppointment sheet

MaximMacarone committed:
Merge branch 'develop' into INTEAM-26

MaximMacarone committed:
Merge pull request #33 from iOS-vk-education/INTEAM-26

Inteam 26
